EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -588,8 +584,6 @@ void HcalUnpacker::unpackUTCA(const FEDRawData& raw, const HcalElectronicsMap& e
     int nps=(amc13->AMCId(iamc)>>12)&0xF;
     
     HcalUHTRData uhtr(amc13->AMCPayload(iamc),amc13->AMCSize(iamc));
-    //Check to make sure uMNio is not unpacked here
```

Comment:
For now, i reverted the changes manually because i think there is something weird going on with my local branch. After this PR is done, i will choose a new local branch with a more creative name